src/universal-dapp.js

```diff
@@ -650,6 +666,7 @@ UniversalDApp.prototype.runTx = function (data, args, cb) {
 
   var tx;
   if (!self.vm) {
+    self.transactionDebugger.switchProvider('EXTERNAL');
```

Comment:
Just to avoid making executionContext dependent of the debugger.
I thought to put the addProvider('VM') and addProvider('EXTERNAL') function in app.js but at that time the vm is not instanciated. so maybe better to setup everything on the same place.
But I can change that if you think it's clearer.